pkg/env/env_test.go

```diff
@@ -0,0 +1,168 @@
+// Copyright The Shipwright Contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+package env
+
+import (
+	"reflect"
+	"testing"
+
+	corev1 "k8s.io/api/core/v1"
+)
+
+func TestMergeEnvVars(t *testing.T) {
```

Comment:
Your test case only tests with `value` but not with `valueFrom`. I think it makes sense to also have it covered.

Reply:
updated in next push